tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -182,9 +190,36 @@ def test_request_makes_vetext_call(mocker, monkeypatch, all_path_env_param_set,
 
     sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_retry_sqs')
     mocker.patch(f'{LAMBDA_MODULE}.read_from_ssm', return_value='ssm')
-    mocker.patch(f'{LAMBDA_MODULE}.requests.post', return_value=mocked_requests_post_success())
+    mock_requests = mocker.patch(f'{LAMBDA_MODULE}.requests.post', return_value=mocked_requests_post_success())
```

Comment:
I'm not insisting on changes for this, but notification-api uses [requests-mock](https://requests-mock.readthedocs.io/en/latest/overview.html).  It would probably be less code.

Reply:
I'll keep requests-mock in mind for future work because it does improve readability, but I would have to rewrite all of the tests in this module to use it.